resources/views/appeals/appeal.blade.php

```diff
@@ -36,73 +36,83 @@
 
-                                <a href="https://meta.wikimedia.org/wiki/Special:CentralAuth?target={{$info['appealfor']}}"
+                                <a href="https://meta.wikimedia.org/wiki/Special:CentralAuth?target={{$info->appealfor}}"
                                    class="btn btn-light">
                                     Global (b)locks
                                 </a>
 
                                 @if($perms['admin'])
-                                    <a href="https://en.wikipedia.org/wiki/Special:Unblock/{{$info['appealfor']}}"
+                                    <a href="https://en.wikipedia.org/wiki/Special:Unblock/{{$info->appealfor}}"
                                        class="btn btn-warning">
                                         Unblock
                                     </a>
                             @endif
                             @if($perms['checkuser'])
                                 <h5 class="card-title">CU data</h5>
                                 @if($checkuserdone)
-                                    IP address: {{$cudata['ipaddress']}}<br/>
-                                    Useragent: {{$cudata['useragent']}}<br/>
-                                    Browser Language: {{$cudata['language']}}
+                                    IP address: {{$cudata->ipaddress}}<br/>
+                                    Useragent: {{$cudata->useragent}}<br/>
+                                    Browser Language: {{$cudata->language}}
                                 @else
                                     <div class="alert alert-danger" role="alert">
                                         You have not submitted a request to view the CheckUser data yet.
                                     </div>
-                                    {{ Form::open(array('url' => 'appeal/checkuser/'.$id)) }}
-                                    {{Form::token()}}
-                                    {{Form::label('reason', 'Reason:')}}<br>
-                                    {{Form::textarea('reason',null,['rows'=>2])}}<br><br>
-                                    <button type="submit" class="btn btn-success">Submit</button>
-                                    {{ Form::close() }}
+                                    <form method="POST" action="/appeal/checkuser/{{ $id }}">
+                                        @csrf
+
+                                        <div class="form-group">
+                                            <label for="cu-reason">
+                                                Reason
+                                            </label>
+
+                                            <textarea name="reason" id="cu-reason" class="form-control"></textarea>
+                                        </div>
+
+                                        <button type="submit" class="btn btn-success">Submit</button>
+                                    </form>
```

Comment:
@supertassu Is there any practical change here?

Reply:
Only practical change here is to wrap the input in `.form-group` and make text area `.form-control`